### ISSUE TITLE
MantisBT: Application error #401

### ISSUE THREAD

**Issue by ghost** [SynoCommunity/spksrc#2276]
Issue description: Application error when loading the website after first time install. After clicking on the URL from the Package Centre (http://diskstation/mantisbt/login_page.php) the following SQL error occurs:

```
APPLICATION ERROR #401
Database query failed. Error received from database was #1146: Table 'mantisbt.mantis_user_table' doesn't exist for the query: SELECT *
FROM mantis_user_table
WHERE username=?.

Please use the "Back" button in your web browser to return to the previous page. There you can correct whatever problems were identified in this error or select another action. You can also click an option from the menu bar to go directly to a new section.
```

Within phpMyAdmin, there is a mantisbt database, however it's empty. I'm guessing the package installer didn't create the necessary database objects? 

Model: DS213air
Architecture: 88f6282 (88f628x) - x13 Series 
DSM version: DSM 6.0-7321 Update 2
Log file: No such folder /usr/local/mantisbt/var/

**Comment by Dr-Bean** [comment]
Confirmed. Easy to fix, there's a `php` entry in the installer that I overlooked.

**Comment by Dr-Bean** [comment]
Done with ac2721812e577480dbbf97ef57c51ac077b40edb. Published mantisbt 1.2.19-5.
